### PR TITLE
Add Booking icon & Link Lineage Music and Newpipe Nightly

### PR DESCRIPTION
**Pull request by mustafakhalaf-git**
## Description

https://play.google.com/store/apps/details?id=com.booking
https://github.com/TeamNewPipe/NewPipe/
https://github.com/LineageOS/android_packages_apps_Etar
https://org-lineageos-eleven.en.uptodown.com/android

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* Booking.com (`com.booking`)

### Icons linked
* NewPipe Nightly (linked `org.schabi.newpipe` to `@drawable/newpipe`)
* Music (linked `org.lineageos.eleven` to `@drawable/mi_music`)

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -794,6 +797,7 @@
     <icon drawable="@drawable/proddy" package="co.thomasmenzel.proddy" name="Proddy" />
     <icon drawable="@drawable/progressbar95" package="com.spookyhousestudios.progressbar95" name="Progressbar95" />
     <icon drawable="@drawable/proton_calendar" package="me.proton.android.calendar" name="Proton Calendar" />
+    <icon drawable="@drawable/proton_calendar" package="org.lineageos.etar" name="Calendar" />
```
Comment (daywalk3r666) [inline app/src/main/res/xml/grayscale_icon_map.xml:800]: Calendar icons are now stored in lawnchair launcher

**Review by daywalk3r666 — CHANGES_REQUESTED** [review]

File: app/src/main/res/xml/grayscale_icon_map.xml (hunk 1 above)
Comment (mustafakhalaf-git) [reply app/src/main/res/xml/grayscale_icon_map.xml:800]: didn't work with me

Comment (daywalk3r666) [reply app/src/main/res/xml/grayscale_icon_map.xml:800]: https://github.com/LawnchairLauncher/lawnchair/commit/aa251677483fde500a0f34fd31e761bdb44566c7

Comment (Chefski) [reply app/src/main/res/xml/grayscale_icon_map.xml:800]: Could you remove the calendar link please?

Comment (mustafakhalaf-git) [reply app/src/main/res/xml/grayscale_icon_map.xml:800]: what do I replace it with? @Chefski

Comment (mustafakhalaf-git) [reply app/src/main/res/xml/grayscale_icon_map.xml]: done, removed

**Review by Chefski — APPROVED** [review]
Thanks

**Pull request merged**